### DISCUSSION TITLE
database wide history of data changes

### DISCUSSION THREAD

**Discussion by Simplenuity** [keepassxreboot/keepassxc#9995] [Ideas]
Would it be possible to implement the option to review all data changes since opening the database?

It's already possible to review the history of a single entry. But that requires the user to know which entry has been edited AND the user would need to open each single edited entry separately, and still making it impossible to discover accidential data changes.

If the history of data changes that occurred since opening the database could be consolidated in one list, it would enable the user to:
* review all the changes before saving the database
* discover and prevent data changes that occurred by accident

Thank you for your consideration

**Comment by droidmonkey** [comment]
See #5465. I thought we had a better issue open then that but I couldn't find it. 

You can also search for `*` and sort by Last Modified to see which entries have changed.

**Comment by Simplenuity** [comment]
Thanks, @droidmonkey! Great to hear it's on your radar/wish/todo list.
Will close this discussion now as it's covered in #5465
